pulp_ansible/tests/functional/api/collection/test_consume_content.py

```diff
@@ -35,7 +35,9 @@ def test_consume_content(self):
         repo = self.client.post(REPO_PATH, gen_repo())
         self.addCleanup(self.client.delete, repo["_href"])
 
-        body = gen_ansible_remote(url=ANSIBLE_GALAXY_COLLECTION_URL, whitelist=COLLECTION_WHITELIST)
+        body = gen_ansible_remote(
+            url=ANSIBLE_COLLECTION_FIXTURE_URL, whitelist=COLLECTION_WHITELIST
```

Comment:
Is the whitelist feature still working with this PR? Now that this PR will provide filtering from the URL iteslf I think the whitelist feature could even be removed. @daviddavis what do you think?